src/main/scala/dpla/ingestion3/model/package.scala

```diff
@@ -248,7 +249,7 @@ package object model {
         |   | permission = {{${permissions}}}
         |   | source = {{ DPLA
         |       | ${escapeWikiChars(dataProviderWikiUri)}
-        |       | hub = ${escapeWikiChars(record.provider.name.getOrElse(""))}
+        |       | ${escapeWikiChars(providerWikiUri)}
```

Comment:
Should still have `hub = ` here

Reply:
Got it, thanks. I assumed it should have looked like the dataProviderWikiId w/o the `hub = ` bit. Will revise.